sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/TracerProviderConfiguration.java

```diff
@@ -53,11 +55,17 @@ static void configureTracerProvider(
         SpanExporterConfiguration.configureSpanExporters(
             config, spiHelper, spanExporterCustomizer, closeables);
 
-    configureSpanProcessors(config, exportersByName, meterProvider, closeables)
-        .forEach(tracerProviderBuilder::addSpanProcessor);
+    List<SpanProcessor> processors =
+        configureExportingSpanProcessors(config, exportersByName, meterProvider, closeables);
```

Comment:
Let's apply the spanProcessorCustomizer to each span processor individually, rather than to the composite processor.